elements/lisk-codec/src/codec.ts

```diff
@@ -67,6 +67,15 @@ export const validateSchema = (schema: {
 		throw new LiskValidationError([...errors]);
 	}
 
+	const { properties } = schemaToValidate as ValidatedSchema;
+	if (properties) {
+		Object.values(properties).forEach(property => {
+			if (!('fieldNumber' in property)) {
```

Comment:
field number can exist in the nested properties, too. 
Did you consider updating `elements/lisk-validator/src/lisk_meta_schema.ts` which is executed in above L63?